### PR TITLE
Latest => v4.0

### PR DESCRIPTION
**Pull request by balexey88**
* NEW - added `Info` section to the `Status` tab on the Settings page, which contains the system info and the ability to copy report to clipboard.  
* ENHANCEMENT - added `Documentation` link on the Plugins page.
* ENHANCEMENT - added `Addons` link on the Plugins page.
* ENHANCEMENT - added `Documentation` link on the Settings page.
* FIX - fixed `Settings` shortcut on the Plugins page.
* FIX - in multisite network, do not show Data Optimization on the Network Admin Page.
* FIX - properly set `Content Disposition` fields for media objects.
* FIX - properly use `Cache Control` setting for media objects.
* FIX - fixed `Creation of dynamic property` PHP deprecation notice.
* FIX - fixed `Cannot use ::class with dynamic class name` PHP warning.
* FIX - avoid PHP warning when unable to get file path in `Stateless` mode [728](https://github.com/udx/wp-stateless/issues/728).
* FIX - fixed links to the constants documentation.

### CODE REVIEW

**Review by kavaribes — APPROVED** [review]

File: lib/classes/status/class-info-stateless.php [hunk 1]
```diff
@@ -0,0 +1,393 @@
+<?php
+/**
+ * System Info (Stateless section) class
+ *
+ * @since 4.1.0
```
Comment (planv) [inline lib/classes/status/class-info-stateless.php:5]: @balexey88 since `4.0.3`

File: lib/classes/status/class-info.php [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * Migrations status class
+ * System Info class
  *
- * @since 4.0.0
+ * @since 4.1.0
```
Comment (planv) [inline lib/classes/status/class-info.php:5]: @balexey88 4.0.3

File: lib/classes/status/class-info-stateless.php (hunk 1 above)
Comment (balexey88) [reply lib/classes/status/class-info-stateless.php:5]: @planv fixed in 4.1.0, thanks!

File: lib/classes/status/class-info.php (hunk 2 above)
Comment (balexey88) [reply lib/classes/status/class-info.php:5]: @planv fixed in 4.1.0, thanks!

**Pull request merged**